event-driven-architecture/src/main/java/com/iluwatar/eda/App.java

```diff
@@ -0,0 +1,41 @@
+package com.iluwatar.eda;
+
+import com.iluwatar.eda.event.Event;
+import com.iluwatar.eda.event.UserCreatedEvent;
+import com.iluwatar.eda.event.UserUpdatedEvent;
+import com.iluwatar.eda.handler.UserCreatedEventHandler;
+import com.iluwatar.eda.handler.UserUpdatedEventHandler;
+import com.iluwatar.eda.model.User;
+
+/**
+ * An event-driven architecture (EDA) is a framework that orchestrates behavior around the
```

Comment:
It has striking similarity with Reactor pattern. I think we can say that Reactor pattern is based on EDA. Can you confirm this?